docs/clients/python.mdx

```diff
@@ -0,0 +1,180 @@
+---
+title: Python
+icon: python
+---
+
+import MvpWarning from "/snippets/mvp-warning.mdx";
+import StepDefineActor from "/snippets/step-define-actor.mdx";
+import StepRunStudio from "/snippets/step-run-studio.mdx";
+import StepDeploy from "/snippets/step-deploy.mdx";
+import SetupNextSteps from "/snippets/setup-next-steps.mdx";
+
+The ActorCore Python client provides a way to connect to and interact with actors from Python applications.
+
+<MvpWarning />
+
+## Quickstart
+
+<Steps>
+  <Step title="Create a new Python project">
+    Create a new directory for your project:
+    
+    ```sh
+    mkdir my-app
+    cd my-app
+    ```
+
+    It's recommended to create a virtual environment:
+    ```sh
+    python -m venv venv
+    source venv/bin/activate  # On Windows use: venv\Scripts\activate
+    ```
+  </Step>
+
+  <Step title="Add dependencies">
+    Install the ActorCore client package:
+    
+    ```sh
+    pip install actor-core-client
+    ```
+  </Step>
+
+  <StepDefineActor />
+
+  <Step title="Create your client">
+    Create a new file `main.py`:
+
+    <CodeGroup>
```

Comment:
Just do Async & Sync examples using events. Add a section below about subscribing to events with callbacks vs receive